### PR TITLE
new issue format

### PR DESCRIPTION
**Pull request by youtous**
This is a proposal for updating issue templates. Inspired by https://github.com/DNSCrypt/dnscrypt-proxy.

Features:
- issues categories: bug, question, feature request
- default labels on issues

This is a draft, feel free to contribute ;)

Github doc about new templates formats: https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

### CODE REVIEW

**Comment by youtous** [comment]
In complement, we could install probot-stale https://github.com/probot/stale in order to cleanup frozen issues. This kind of app helps for keeping opened only issues that have a significant impact on the user experience.

File: .github/ISSUE_TEMPLATE/questions.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: "❓ Question"
+about: Ask a question about dnscrypt-proxy or the DNSCrypt protocol
```
Comment (erik-wramner) [inline .github/ISSUE_TEMPLATE/questions.md:3]: Should not be DNSCrypt here right?

Comment (youtous) [reply .github/ISSUE_TEMPLATE/questions.md]: :sweat_smile: yes!

**Comment by youtous** [comment]
@erik-wramner Do you have permissions to add the following labels: `needs triage`, `feature request`?

**Comment by erik-wramner** [comment]
Done

**Pull request merged**